custom_components/elmo_iess_alarm/binary_sensor.py

```diff
@@ -56,7 +58,7 @@ def __init__(
         self._unique_id = unique_id
         self._sensor_id = sensor_id
         self._sensor_type = sensor_type
-        self._name = f"{DOMAIN} {name}"
+        self._name =name
```

Comment:
I think you may need to run pre-commit hooks to verify that the code is properly linted. I've added some instructions here: https://github.com/palazzem/ha-econnect-alarm#dev-environment

Let me know if you need help in using pre-commit hooks!